internal/cli/organizations.go

```diff
@@ -212,11 +213,11 @@ auth0 orgs create --n myorganization -d "My Organization" -m "KEY=value" -m "OTH
 				o.Branding = &management.OrganizationBranding{}
 
 				if isLogoURLSet {
-					o.Branding.LogoUrl = &inputs.LogoURL
+					o.Branding.LogoURL = &inputs.LogoURL
 				}
 
 				if isAnyColorSet {
-					o.Branding.Colors = map[string]string{}
+					o.Branding.Colors = map[string]interface{}{}
```

Comment:
This has also changed within the Go SDK, however the underlying field values remain strings. That's why further down we cast again to `.(string)`.